CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+If you don't have git on your machine, [install it](https://help.github.com/articles/set-up-git/).
+
+## The beginner's guide to contributing to a GitHub project
+
+<img align="right" width="300" src="assets/images/fork.png" alt="fork this repository" />
+
+## Fork this repository
+
+Fork this repository by clicking on the fork button on the top of this page.
+This will create a copy of this repository in your account.
+
+## Clone the repository
+
+<img align="right" width="300" src="assets/images/clone.png" alt="clone this repository" />
+
+Now clone the forked repository to your machine. Go to your GitHub account, open the forked repository, click on the code button and then click the _copy to clipboard_ icon.
+
+Open a terminal and run the following git command:
+
+```
+git clone "url you just copied"
+```
+
+where "url you just copied" (without the quotation marks) is the url to this repository (your fork of this project). See the previous steps to obtain the url.
+
+<img align="right" width="300" src="assets/images/url.JPG" alt="copy URL to clipboard" />
+
+For example:
+
+```
+git clone https://github.com/this-is-you/Fike.git
+```
+
+where `this-is-you` is your GitHub username. Here you're copying the contents of the first-contributions repository on GitHub to your computer.
+
+## Create a branch
+
+Change to the repository directory on your computer (if you are not already there):
+
+```
+cd Fika
+```
+
+Now create a branch using the `git checkout` command:
+
+```
+git checkout -b <add-your-new-branch-name>
+```
+
+For example:
+
+```
+git checkout -b add-alonzo-church
+```
+
+(The name of the branch does not need to have the word _add_ in it, but it's a reasonable thing to include because the purpose of this branch is to add your name to a list.)
+
+## Make necessary changes and commit those changes
+
+Now open `Contributors.md` file in a text editor, add your name to it. Don't add it at the beginning or end of the file. Put it anywhere in between. Now, save the file.
```

Comment:
@emilsoman are we keeping this? We don't have a Contributors.md file as of now
If we are keeping it, let's make one and add Emil and yourselves 🎉

Reply:
Ok i'll add Emil sir and me as contributor